appserver/tests/quicklook/run_test.sh

```diff
@@ -90,6 +90,19 @@ run_test_id(){
 		exit 1
 	fi
     change_junit_report_class_names
+}
+
+post_test_run(){
+    if [[ $? -ne 0 ]]; then
+    	if [[ $TEST_ID = "ql_gf_full_profile_all" || $TEST_ID = "ql_gf_web_profile_all" || $TEST_ID = "ql_gf_embedded_profile_all" ]]; then
+	  		copy_ql_results || true
+	  	fi
+	  	if [[ $TEST_ID = "ql_gf_nucleus_all" || $TEST_ID = "nucleus_admin_all" ]]; then
```

Comment:
This run_test.sh is wrapper for the following test ids 
list_test_ids(){
	echo ql_gf_full_profile_all ql_gf_nucleus_all ql_gf_web_profile_all ql_gf_embedded_profile_all nucleus_admin_all
}